src/claimList.js

```diff
@@ -14,38 +25,223 @@ if (document.readyState === "loading") {
 }
 
 function executeOverride() {
-  const allClaims = getMetadata();
-  updateIcon();
   setupAnalytics();
-  logView(allClaims);
+  makeMobileFriendly();
+
+  const { claims, name } = getMetadata();
+  removeOldHtml();
+  addNewHtml(name, claims);
+
+  styleBody();
   addFeedbackLink();
+  updateIcon();
+  logView(claims);
 }
 
 function getMetadata() {
-  return Array.from(
-    document.getElementsByTagName("table")[2]?.children[0]?.children
-  )
+  const tableElements = document.getElementsByTagName("table");
+  const claims = Array.from(tableElements[2]?.children[0]?.children)
     .slice(1)
     .map((row, i) => ({
-      claimType: row.children[1].innerText.trim(),
-      claimDate: row.children[2].innerText.trim(),
-      claimStatus: row.children[3].innerText.trim(),
+      seqNum: row.children[0].innerText.trim(),
+      type: row.children[1].innerText.trim(),
+      date: row.children[2].innerText.trim(),
+      status: row.children[3].innerText.trim(),
     }));
```

Comment:
Dust/Pebble: Very annoying that the original page has so many tables lol.. It feels more readable to just first separately define the table being used to grab the claims from. Also I believe you can use [rows](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTableElement/rows) property to get all the `<tr>` from the table instead rather than having to do `children[0]?.children`. Also, `i` in the .map method isn't being used.

So i think it could be something like this -> 
```
const tableElements = document.getElementsByTagName("table");
const claimsTable = tableElements[2]
const claims =  Array.from(claimsTable?.rows).slice(1).map(row => ({
    seqNum: row.children[0].innerText.trim(),
    type: row.children[1].innerText.trim(),
    date: row.children[2].innerText.trim(),
    status: row.children[3].innerText.trim(),
  }));
```

I also believe you can use

Reply:
I didn't realize you could do that! This definitely feels cleaner, I'll make this edit